Sprint-3/reading-list/script.js

```diff
@@ -21,3 +21,25 @@ const books = [
   },
 ];
 
+function readingList() {
+  const unorderedList = document.querySelector("#reading-list");
```

Comment:
Is this a good name for the variable? Another thing, what if you code cannot find this dom element? How can you guard your function against it?